draco/asp/soft.lp

```diff
@@ -268,31 +279,35 @@ preference(color_entropy_high,E) :-
     attribute((encoding,field),E,F),
     attribute((encoding,channel),E,color),
     helper((encoding,scale_type),E,linear),
-    EN > 5000.
+    helper(enc_interesting,E),
+    EN > 12000.
```

Comment:
Why did you change these values?

Reply:
the updated one is the same as in draco1. The previous values was written by @Zehua-Zeng and the reason I remembered was that she observed in the kim2018 dataset, the entropy values never exceeded 12000? I changed it back to check the testing score for draco-learn.

Reply:
Okay. I don't think the prior values were super thought out so your values may be better.